SNUTT-2022/SNUTT/Views/Components/WebViews/SyllabusWebView/WebProgressView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  WebProgressView.swift
+//  SNUTT
+//
+//  Created by 박신홍 on 10/19/24.
+//
+
+import UIKit
+
+final class WebProgressView: UIView {
+    private let progressBar = UIView()
+    private var progress: Double = 0 {
+        didSet {
+            updateProgressBarFrame()
+        }
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+
+    @available(*, unavailable) required init?(coder _: NSCoder) {
+        fatalError()
+    }
+
+    private func setupView() {
+        progressBar.backgroundColor = .systemBlue.withAlphaComponent(0.9)
+        progressBar.layer.shadowColor = UIColor.blue.cgColor
+        progressBar.layer.shadowRadius = 10.0
+        progressBar.layer.shadowOpacity = 0.9
```

Comment:
그라데이션까지 넣으니까 좀 튀어서 그림자는 뺄까 싶은데 어때
```suggestion
        progressBar.layer.shadowColor = UIColor.blue.withAlphaComponent(0.5).cgColor
```

https://github.com/user-attachments/assets/3674fe00-4a9c-4276-8580-2463801d9976

Reply:
`progressBar.layer.shadowColor = UIColor.blue.withAlphaComponent(0.5).cgColor` 이건 뭐야? 이것도 없애자는거 맞지?

Reply:
앗 깃헙 코멘트 알림이 왜 안떴지...그림자 아예 빼자는거였는데 suggestion 잘못 입력한듯